installation/citus_cloud.rst

```diff
@@ -1,23 +1,18 @@
 .. _multi_node_cloud:
 
-Managed Deployment
+Managed Service
 ==================
 
 One of the easiest ways to get started with Citus is to spin up a cluster in
-the cloud on Azure. Citus is available as a built-in deployment option in the
-`Azure Database for PostgreSQL - Hyperscale (Citus)
-<https://docs.microsoft.com/azure/postgresql/hyperscale/>`_ managed
-service.
+the cloud on Azure. The Citus managed service is available as
+`Azure Cosmos DB for PostgreSQL
+<https://learn.microsoft.com/azure/cosmos-db/postgresql/introduction/>`_, formerly known as Hyperscale (Citus) in Azure Database for PostgreSQL.
 
-To get started, you can take advantage of our Hyperscale (Citus) `quickstart
+To get started, you can take advantage of our Citus on Azure `quickstart
 documentation
-<https://docs.microsoft.com/azure/postgresql/quickstart-create-hyperscale-portal>`_,
+<https://learn.microsoft.com/azure/cosmos-db/postgresql/quickstart-create-portal>`_,
 or go straight to the `Azure Portal
-<https://portal.azure.com/#create/Microsoft.PostgreSQLServer>`_ to provision a
-Postgres server. (And if you don’t yet have an Azure subscription, just `create
-a free Azure account <https://azure.microsoft.com/free/>`_ first.)
+<https://portal.azure.com/#view/Microsoft_Azure_DocumentDB/CreatePostgreSQL.ReactView>`_ to provision an
+Azure Cosmos DB for PostgreSQL cluster. (And if you don't yet have an Azure account, just `create
+a free Azure account <https://azure.microsoft.com/free/>`_—or take advantage of the `free trial <https://cosmos.azure.com/try/>`_.)
```

Comment:
Rather than all these options, what if we sent them to just the free trial? Sounds like it would be the most enticing option if I were a new user.